reactDashboard/src/App.css

```diff
@@ -1,83 +1,167 @@
-.underHeatmapSelector {
-  border: 1px solid #2a84bb;
-  margin-top: 8px;
-  margin-bottom: 8px;
+.darkmode {
+  background-color: #121212;
 }
 
-.full-width-hr {
-  border: 1px solid #2a84bb;
-  margin: 10px 0;
-  padding: 0;
+.full-screen {
+  background-color: white;
 }
 
-.full-screen {
-  background-color: White;
+/* color is default white for header */
+header {
+  padding-top: 10px;
+}
+
+.darkmode header {
+  background-color: #121212;
+}
+
+/* color is default white for .region-selector-label */
+.darkmode .region-selector-label {
+  color: #f2f2f2;
+}
+
+.heatmaps-row,
+.region-page-top-row {
+  padding: 15px 10px;
+  background-color: #f2f2f2;
+  /* required to keep the alignment down left and right hand side of pages */
+  margin: 0;
+}
+
+.darkmode .heatmaps-row {
+  background-color: #262626;
 }
 
-.link {
-  color: #3c3c3b;
+.selector-group fieldset legend {
+  font-size: 16px;
   font-weight: bold;
-  text-decoration: none;
 }
 
+.darkmode .selector-group fieldset legend {
+  color: #f2f2f2;
+}
+
+.darkmode .region-page-top-row {
```

Comment:
Move this block under the block it overrides please

Reply:
✅

Reply:
moved